tools/vagrant/README.md

```diff
@@ -5,6 +5,7 @@ The following instructions were tested on Mac OS X El Capitan, Ubuntu 14.04.3 LT
 *Requirements*
 - Install [VirtualBox](https://www.virtualbox.org/wiki/Downloads) (tested with version 5.1.22)
 - Install [Vagrant](https://www.vagrantup.com/downloads.html) (tested with version 1.9.5)
+- Install [Cygwin64 with ssh](https://www.cygwin.com) (tested with verstion 2.8.0)
```

Comment:
It was unintended. I made a new commit retracting this from the Readme file. Thanks!